src/FillArrays.jl

```diff
@@ -15,6 +17,7 @@ import Base.Broadcast: broadcasted, DefaultArrayStyle, broadcast_shape
 
 export Zeros, Ones, Fill, Eye
 
+""" `AbstractFill{T, N, Axes} <: AbstractArray{T, N}` """
```

Comment:
This docstring is not useful so either delete it, or change it to something like
```
AbstractFill{T, N, Axes} <: AbstractArray{T, N}

Supertype for lazy array types whose entries are all equal to constant.
```

Reply:
Oops, that was pretty useless.